data_management/tests/test_api.py

```diff
@@ -789,3 +790,341 @@ def test_filter_by_key(self):
+                "prov:entity": "api/data_product/2",
+                "prov:agent": "api/author/3",
+                "prov:role": "author",
+            },
+            "_:id9": {
+                "prov:entity": "api/data_product/1",
+                "prov:agent": "api/author/1",
+                "prov:role": "author",
+            },
+            "_:id12": {
+                "prov:entity": "api/data_product/4",
+                "prov:agent": "api/author/2",
+                "prov:role": "author",
+            },
+            "_:id15": {
+                "prov:entity": "api/data_product/5",
+                "prov:agent": "api/author/3",
+                "prov:role": "author",
+            },
+            "_:id19": {
+                "prov:entity": "api/data_product/3",
+                "prov:agent": "api/author/3",
+                "prov:role": "author",
+            },
+        }
+        self.assertEqual(results["wasAttributedTo"], expected_result)
+
+        expected_result = {
+            "_:id11": {
+                "prov:generatedEntity": "api/data_product/2",
+                "prov:usedEntity": "api/data_product/1",
+            },
+            "_:id14": {
+                "prov:generatedEntity": "api/data_product/2",
+                "prov:usedEntity": "api/data_product/4",
+            },
+            "_:id17": {
+                "prov:generatedEntity": "api/data_product/2",
+                "prov:usedEntity": "api/data_product/5",
+            },
+        }
+        self.assertEqual(results["wasDerivedFrom"], expected_result)
+
+        expected_result = {
+            "_:id3": {
+                "prov:entity": "api/data_product/2",
+                "prov:activity": "api/code_run/1",
+            },
+            "_:id20": {
+                "prov:entity": "api/data_product/3",
+                "prov:activity": "api/code_run/1",
+            },
+        }
+        self.assertEqual(results["wasGeneratedBy"], expected_result)
+
+        expected_result = {
+            "_:id4": {
+                "prov:activity": "api/code_run/1",
+                "prov:trigger": "api/user/1",
+                "prov:time": "2021-07-17T18:21:11+00:00",
+                "prov:role": "code runner",
+            }
+        }
+        self.assertEqual(results["wasStartedBy"], expected_result)
+
+    def test_get_provn(self):
+        client = APIClient()
+        client.force_authenticate(user=self.user)
+        url = reverse("prov_report", kwargs={"pk": 1})
+        response = client.get(
+            url, format="provn", HTTP_ACCEPT="text/provenance-notation"
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response["Content-Type"], "text/provenance-notation; charset=utf8"
+        )
+
+        result_bits = response.data.split("last_updated")
+        result_end = result_bits[1].split("xsd:dateTime, ", 1)[1]
+        result = result_bits[0] + result_end
+        expected_result = """document
```

Comment:
This is no longer valid, as it contains a hardwired URL involving https://data.scrc.uk/

Reply:
https://github.com/FAIRDataPipeline/data-registry/pull/128/commits/dead28c0e3c3913e5fa1a17cf0340b3b9b0208bc